PROVISIONING-METAL.md

```diff
@@ -77,8 +77,8 @@ When these services fail, your machine will not connect to any cluster and will
 #### `net.toml` structure
 
 The configuration file must be valid TOML and have the filename `net.toml`.
-The first and required top level key in the file is `version`; the latest is version `2`.
-The rest of the file is a map of interface name to supported settings.
+The first and required top level key in the file is `version`; the latest is version `3`.
```

Comment:
This will all drastically change when I rebase on the bonding/vlan changes.

Just looking to get eyes on wording, etc.